src/CloudNative.CloudEvents.AspNetCore/CloudEventInputFormatter.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Cloud Native Foundation. 
+// Licensed under the Apache 2.0 license.
+// See LICENSE file in the project root for full license information.
+
+namespace CloudNative.CloudEvents
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Mvc.Formatters;
+    using Microsoft.Net.Http.Headers;
+
+    public class CloudEventInputFormatter : TextInputFormatter
```

Comment:
suggestion: should this be called `CloudEventJsonInputFormatter`?

Reply:
Yes, renamed the formatter, and changed the supported media type to the more specific `application/cloudevents+json`.